spec/factories/user.rb

```diff
@@ -15,5 +15,6 @@ def generate_jwt_token(user)
     last_name { FFaker::NameDE.last_name }
     email { FFaker::Internet.email }
     password { "Hh4#{Random.hex(7)}" }
+    status { 1 }
```

Comment:
in my opinion, maybe best way for a factory name `:admin` because it's not a magic number

Reply:
agree